### PR TITLE
Include git metadata in code locations by default

### PR DESCRIPTION
**Pull request by benpankow**
## Summary

Includes git metadata (commit SHA and backlink) for code locations created by the action by default. Can be disabled by setting `include-git-metadata` to false.

## Test Plan

Existing examples now have metadata.

### CODE REVIEW

**Comment by benpankow** [comment]
Current dependencies on/for this PR:
* main
  * **PR #9** Graphite  (this PR)

This comment was auto-generated by [Graphite](https://graphite.dev?utm_source=gh).

**Comment by rexledesma** [comment]
Does this need to be configurable by a flag? Why wouldn't users just want this all the time?

**Review by rexledesma — CHANGES_REQUESTED** [review]
I believe this should just be on by default, and not be configurable

File: src/deploy-action.js [hunk 1]
```diff
@@ -49,9 +49,8 @@ async function run() {
           );
         }
 
-        // Optionally include some experimental git data in the location metadata
+        // Optionally include some git data in the location metadata
```
Comment (rexledesma) [inline src/deploy-action.js:52]: can get rid of this comment

**Review by rexledesma — APPROVED** [review]

**Pull request merged**